src/components/atoms/dropdown-v2/dropdown.types.ts

```diff
@@ -1,7 +1,7 @@
 import { FormGroup } from '../../../core/form/useForm/useForm.types';
 
 export type DropdownItem = {
-  id: string | number;
+  id?: string | number;
```

Comment:
The fact that you made id optional is okey, just make sure the values you are passing to dropdonw are unique as they are used for the `key` prop.

We used to have the issue where values received from API were not unique.

Reply:
in some sections, you didn't pass ids (static data) and the typescript gave a warning. 
we have 2 options: 1) make it optional and remember the unique id for API data and 2) change all static data and constants and add `id` properties to their data.
whatever you decide.